kms/api-client/snippets_test.py

```diff
@@ -177,10 +178,18 @@ def test_key_policy(self):
                                                  self.symId,
                                                  self.member,
                                                  self.role)
-        policy = snippets.get_crypto_key_policy(self.project_id,
-                                                self.location,
-                                                self.keyring_id,
-                                                self.symId)
+        try_number = 0
+        policy = None
+        while policy is None and try_number < self.try_limit:
+            try:
+                time.sleep(2*try_number)
+                policy = snippets.get_crypto_key_policy(self.project_id,
+                                                        self.location,
+                                                        self.keyring_id,
+                                                        self.symId)
+            except Aborted:
+                # aborted by backend. Try again
+                try_number += 1
```

Comment:
Maybe you want to use `@eventually_consistent.call`?

An example commit:
https://github.com/GoogleCloudPlatform/python-docs-samples/pull/3253/commits/c6254e46bca355ce422fbd4844fd1928f37b89c0

Reply:
+1 to using eventually consistent if you can

Reply:
I didn't know about `@eventually_consistent`, that's much cleaner!